charts/scalardb/files/scalardb_grafana_dashboard.json

```diff
@@ -79,7 +79,7 @@
       "targets": [
```

Comment:
These changes about Scalar DB dashboard is expected?  (I was expecting the changes in charts of scalardl and scalardl-audit from the PR title)
If it is not related, it would be great if this is a separate PR. (the changes look good!)

Reply:
@feeblefakie 
Sorry.
I had mentioned it in the description.
```
Fixed the dashboard because the scalardb metrics name changed.
```

The title was confusing, so I have corrected the title.
https://github.com/scalar-labs/helm-charts/pull/24#event-5269350612

Reply:
@miseyu Oh, got it. Thank you!